src/cascadia/TerminalConnection/ConptyConnection.h

```diff
@@ -4,16 +4,19 @@
 #pragma once
 
 #include "ConptyConnection.g.h"
-// Note that the ConptyConnection is no longer a part of this project
-// Until there's platform-level support for full-trust universal applications,
-// all ProcThreadAttribute things will be unusable. Unfortunately, this means
-// we'll be unable to use the conpty API directly.
+
+namespace wil
+{
+    // These belong in WIL upstream, so when we reingest the change that has them we'll get rid of ours.
```

Comment:
Naked TODO, leave WI.

Reply:
This one is self-resolving because it will turn into a build break when wil has a unique_pseudoconsole_handle.

Reply:
OK OK.